content/posts/android-tv-12.md

```diff
@@ -0,0 +1,142 @@
+Today we're bringing the Android TV app to 2021 with a modern design and tons of other changes.
+
+<!--more-->
+
+Over 400 [^project] pull requests containing 2400+ commits, 750+ changed files with 54000+ changed lines of code by roughly 50 contributors[^git-diff]. This is the biggest update to the app we ever worked on. I'll talk a bit about why it is this big and why it took so long later in the post. But let's start with the part you probably came for: **new features!**
+
+[^project]: [v0.12.0 project filtered by merged pull requests](https://github.com/jellyfin/jellyfin-androidtv/projects/2?card_filter_query=is%3Apr+is%3Amerged)
+[^git-diff]: [Git comparison between v0.11.5 and v0.12.0-beta.7](https://github.com/jellyfin/jellyfin-androidtv/compare/v0.11.5...v0.12.0-beta.7)
+
+## New authentication flow
+
+We got a lot of complaints about our authentication flow. It wasn't obvious how to enable the auto log in option and managing multiple servers or users was not an easy task. We also didn't like this part of the app, and decided to completely revamp it! The rewritten sign in screen looks more modern and is much easier to use. There is a new help section that links to our documentation to help new users getting started with Jellyfin. The auto-discovery feature of the app now shows all servers instead of the first one, you can select them to connect or manually enter your server address. You can then proceed to add a user and start using the app. Users are automatically saved now with auto log in enabled by default.
+
+<div class="juxtapose">
+    <img data-label="0.11" src="/images/posts/androidtv-0-12-0/11-user-select.png" />
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-user-select.png" />
+</div>
+
+## Updated user interface design
+
+After logging in you will be presented with a modern look and feel in our refreshed user interface. A new toolbar is added to the top-right corner of the screen to open the search page, settings or to switch to a different user. No need to scroll down to the bottom of the home screen anymore.
+Beneath this new toolbar is your media, like it always was, but the cards got a new look with the debatable colored backgrounds removed.
+
+<div class="juxtapose">
+    <img data-label="0.11" src="/images/posts/androidtv-0-12-0/11-home.png" />
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-home.png" />
+</div>
+
+The settings screen got a refresh too. It allows you to more easily change settings. Some new settings got added to customize the app to your own tastes.
+
+<div class="juxtapose">
+    <img data-label="0.11" src="/images/posts/androidtv-0-12-0/11-settings.png" />
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-settings.png" />
+</div>
+
+The user interfacer when browsing inside libraries got some slight changes too. We're hoping to completely revamp this part of the app at some point, but that doesn't stop us from making it slightly better right now.
```

Comment:
Or "didn't" instead of won't if the changes are already in the update.

```suggestion
The user interface when browsing inside libraries got some slight changes too. We're hoping to completely revamp this part of the app at some point, but that won't stop us from making it slightly better now.
```